src/app/communities/page.tsx

```diff
@@ -16,16 +17,21 @@ export default function Communities() {
         <Image
           src="/images/communities-banner.svg"
           alt="Communities"
-          width={0}
-          height={0}
-          style={{ width: '100%', height: 'auto' }}
+          width={2272}
+          height={378}
+          className="w-full h-auto mb-[60px] mt-[60px]"
```

Comment:
You can keep width and height as `0` and remove `h-auto` from `className`. Let me know if it works on your end.

Reply:
Good tip!